guides/development-workflow.md

```diff
@@ -190,6 +190,7 @@ To deploy:
   ```
   git fetch
   git checkout master
+  git pull
```

Comment:
Pixel Tom pointed out that git pull would create merge commits when I originally included `git pull` in this list:

> Incorporates changes from a remote repository into the current branch. In its default mode, git pull is shorthand for git fetch followed by git merge FETCH_HEAD

https://git-scm.com/docs/git-pull

What do you think?